src/components/NavBar/index.tsx

```diff
@@ -97,7 +96,7 @@ export default function NavBar() {
     <Styles.NavBarContainer>
       <Styles.NavBarSectionDiv>
         <Link href="/">
-          <Icon type="logo" />
+          <Icon type="IJPLogo" />
         </Link>
         {navlinks.map(NavLink => renderLink(NavLink))}
```

Comment:
let's put the definition of `renderLink` in here instead of outside the function.
also, remember to add the `key` prop to the outermost wrapper of the render function to avoid this warning:
![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/86812571-3852-4423-9452-7ce684ca08e2)